playwright/playwright.config.ts

```diff
@@ -0,0 +1,86 @@
+import type { PlaywrightTestConfig } from "@playwright/test";
+import { devices } from "@playwright/test";
+
+// See https://playwright.dev/docs/api/class-testconfig
+const config: PlaywrightTestConfig = {
+  // Configuration for the expect assertion library
+  expect: {
+    // Maximum time expect() should wait for the condition to be met. For
+    // example in `await expect(locator).toHaveText();`
+    timeout: 30_000,
+  },
+
+  // Fail the build on CI if you accidentally left test.only in the source code.
+  forbidOnly: !!process.env.CI,
+
+  // Retry on CI only.
+  retries: process.env.CI ? 2 : 0,
+
+  // Configure projects for major browsers.
+  projects: [
+    {
+      name: "onboarding",
+      testMatch: "onboarding.spec.ts",
+    },
+    {
+      name: "chromium",
+      use: {
+        ...devices["Desktop Chrome"],
+      },
+      dependencies: ["onboarding"],
+    },
+  ],
```

Comment:
No, I found it super strange at first also! Still now ^^

<img width="1025" alt="Screenshot 2023-11-27 at 18 41 49" src="https://github.com/prismicio/slice-machine/assets/19946868/8eb251df-1007-4ee5-a502-9563de3eca5f">

![Screenshot 2023-11-27 at 18 42 01](https://github.com/prismicio/slice-machine/assets/19946868/82b4ab64-1527-4193-b357-feea449d3609)

EDIT: But what I can do is merging both a bit more:

Also with the conf to prevent opening automatically on fail.

```
reporter: [
    [
      "html",
      {
        open: "never",
      },
    ],
    ...((process.env.CI ? ["github"] : []) as ReporterDescription),
  ]
```

I have to cast else it don't like it. Or maybe I can just do as before and duplicate the open: never, WDYT?